docs/src/behavior.md

```diff
@@ -36,11 +39,11 @@ The method is called with the following arguments:
 
 - `event`, the actual event triggering the selection.
 
-- `suggestion`, the object representing the current selection. Contains a `.url`
-  key representing the destination.
+- `suggestion`, the object representing the current selection. It contains a
+  `.url` key representing the destination.
 
-- `datasetNumber`, this should always equal `1` as DocSearch is searching into
-  one dataset at a time. You can ignore this attribute.
+- `datasetNumber`, this should always be equal `1` as DocSearch is searching
```

Comment:
Either `will always equal something`, or `will always be equal to something`. Here you either are missing a `to`, or have a `be` that shouldn't be here